manifests/resource/upstream.pp

```diff
@@ -10,6 +10,7 @@
 #   [*upstream_cfg_prepend*]  - It expects a hash with custom directives to put before anything else inside upstream
 #   [*upstream_fail_timeout*] - Set the fail_timeout for the upstream. Default is 10 seconds - As that is what Nginx does normally.
 #   [*upstream_max_fails*]    - Set the max_fails for the upstream. Default is to use nginx default value which is 1.
+#   [*upstream_weight*]       - Set the weight for the upstream. Default is to use nginx default value which is 1.
```

Comment:
hmm... i would have preferred to have just reformulated the doc line. Now you have a change that affects every user. 

And how is one supposed to pass set the param undef? I don't think this is possible because then the default of 1 will slip in again.